MentalHospital.Tests/CrudOperatienTests_PatientService.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Xunit;
+using Moq;
+using AutoMapper;
+using Shouldly;
+using MentalHospital.BLL.Mapper;
+using MentalHospital.BLL.Models;
+using MentalHospital.BLL.Services;
+using MentalHospital.DAL.Interfaces;
+using MentalHospital.DAL.Entities;
+using MentalHospital.Tests.InitialMethods;
+
+
+namespace MentalHospital.Tests
+{
+    public class CrudOperationTests_PatientService
```

Comment:
Just PatientServiceTests, the same for file name
And project name not updated, should be MentalHospital.BLL.Tests